srcpkgs/subsurface/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'subsurface'
+pkgname=subsurface
+version=5.0.5
+revision=1
+archs="i686 x86_64 x86_64-musl aarch64 aarch64-musl"
+wrksrc="Subsurface-${version}"
+build_style=cmake
+configure_args="-DSUBSURFACE_TARGET_EXECUTABLE=DesktopExecutable -DCMAKE_BUILD_TYPE=Release -DBTSUPPORT=ON -DNO_USERMANUAL=OFF -DNO_PRINTING=OFF"
+make_cmd=make
+hostmakedepends="git pkg-config autoconf automake libtool"
+makedepends="libdivecomputer-subsurface-devel libbluetooth-devel libcurl-devel sqlite-devel libssh2-devel eudev-libudev-devel libusb-devel libxml2-devel libxslt-devel qt5-devel qt5-connectivity-devel qt5-declarative-devel qt5-location-devel qt5-script-devel qt5-svg-devel qt5-tools-devel qt5-webkit-devel bluez-qt5-devel libgit2-devel libzip-devel libmtp-devel"
```

Comment:
Break these lines at ~80 columns.

Reply:
Done

Reply:
Breaking configure_args causes xlint to fail